src/WalkingModule/include/WalkingControllers/WalkingModule/Module.h

```diff
@@ -66,6 +70,68 @@ namespace WalkingControllers
         double m_dT; /**< RFModule period. */
         double m_time; /**< Current time. */
         std::string m_robot; /**< Robot name. */
+        double m_stepHeight; /**< maximum height of step. */
+        double m_startOfWalkingTime;  /**< The time that the robot starts walking. */
+
+        double m_timeOffset;/**< timeOffset is the time of start of this step(that will be updated in updateTrajectory function at starting point of each step)*/
+        double m_impactTimeNominal;/**< Nominal absolute time of the impact */
+        double m_impactTimeAdjusted;/**< Adjusted absolute time of the impact */
+        iDynTree::Vector2 m_zmpNominal;/**< Nominal 2D zmp position */
+        iDynTree::Vector2 m_zmpAdjusted;/**< Absolute 2D zmp position */
+        int m_indexPush;/**< Number of control cycle that step adjustment is active */
```

Comment:
The `WalkingModule` is already a class full of parameters please avoid to add other parameters. E.g.. we really need the `m_stepHeight` here?

Reply:
Actually as we discussed T2T we will solve it generally after some other modification inside walking module.